WcaOnRails/app/views/wiki_pages/show.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= provide(:title, "Wiki") %>
+
+<div class="container">
+  <article class="wiki-page">
+    <header class="title">
+      <%= @wiki_page.title %>
+    </header>
+    <main class="content">
+      <%= md @wiki_page.content %>
+    </main>
+    <aside class="info">
+      Created by <%= @wiki_page.author.name %> on <%= @wiki_page.created_at.strftime "%d %b %Y at %R" %>
+      Last update on <%= @wiki_page.updated_at.strftime "%d %b %Y at %R" %>
```

Comment:
Please see discussion here: https://github.com/cubing/worldcubeassociation.org/pull/815#discussion_r77044960.

Basically, I would not like to see us explicitly calling `strftime` anywhere in our codebase.